cfgov/agreements/jinja2/agreements/_json.html

```diff
@@ -0,0 +1,17 @@
+{# ==========================================================================
+
+   Includes issuers as a global JS object
+
+   issuers: List of credit card agreement issuers.
+
+   ========================================================================== #}
+<script>
+  window.ISSUERS = [
```

Comment:
Minor thing, but maybe make this `window.cfpbAgreements = [` or similar, so it follows custom CFPB data attached to the window like https://github.com/cfpb/consumerfinance.gov/blob/abaeafe57c4f6e0f8783f9159e7e6c9cb8a64ddf/cfgov/housing_counselor/jinja2/housing_counselor/index.html#L320